src/services/models/Response.ts

```diff
@@ -8,6 +8,15 @@ import { RedocNormalizedOptions } from '../RedocNormalizedOptions';
 import { FieldModel } from './Field';
 import { MediaContentModel } from './MediaContent';
 
+type ResponseProps = {
+  parser: OpenAPIParser,
+  code: string,
+  defaultAsError: boolean,
+  infoOrRef: Referenced<OpenAPIResponse>,
+  options: RedocNormalizedOptions,
+  reverseEventsReadWriteProps: boolean,
```

Comment:
I don't like this `reverseEventsReadWriteProps`... How about `isWebhookOrCallback` or `isEvent`